<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test</class>
 <widget class="QWidget" name="test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit_reply">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>331</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_ask">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>600</y>
     <width>341</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_send">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>620</y>
     <width>93</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>501</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>我是你的音乐小助手，我叫小果，有什么问题都可以问我哦！</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
